README.md

```diff
@@ -1 +1,105 @@
-# angular-multi-slot-transclusion-readme
\ No newline at end of file
+# Multi-slot transclusion
+
+## Overview
+
+One other awesome thing that transclusion offers is the concept of multi-slot transclusion. This allows us to pick different elements in our transclusded content and put them wherever we like.
+
+## Objectives
+
+- Describe the new multi-slot transclude feature
+- Create multiple transclusion slots
+- Create named transclusion slots
+- Define optional transclusion slots
+
+## transclude: {}
+
```

Comment:
Add some context around why we need this first. What problem is multi-slot transclusion solving for us?